sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -568,11 +568,6 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_adl_machines[] = {
 		.quirk_data = &adl_max98390_amp,
 		.sof_tplg_filename = "sof-adl-max98390-rt5682.tplg",
 	},
-	{
-		.comp_ids = &adl_rt5682_rt5682s_hp,
-		.drv_name = "adl_rt5682",
-		.sof_tplg_filename = "sof-adl-rt5682.tplg",
-	},
```

Comment:
why would you move code that's unrelated to your own work? Just add yours without changing the existing code.

Reply:
Priority and selection order need to be changed otherwise default driver "adl_rt5682" has been loaded instead of "adl_rt5682_c1_h02".  because of the codec ACPI id got matched and didn't check other machines after this machine.
Hence default driver "adl_rt5682 " need to be in last in the list.

Reply:
leave this here and insert your new code where it belongs. Always minimize changes.

Reply:
Agree. will update.